deepchecks/vision/metrics_utils/metrics.py

```diff
@@ -45,18 +49,58 @@ def get_default_object_detection_scorers() -> t.Dict[str, Metric]:
     }
 
 
+_func_naming_dict = {
+    'precision': 'Precision',
+    'recall': 'Recall',
+    'sensitivity': 'Recall',
+    'average_precision': 'Average Precision',
+    'average_recall': 'Average Recall',
```

Comment:
If not too messy, also add mean_average_precision and mean_average_recall options

Reply:
kinda is